### ISSUE TITLE
Ctrl+f and ctrl+b sometimes jump to a postion off-by-one

### ISSUE THREAD

**Issue by mmokrejs** [igvteam/igv#148]
I was moving around the VCF tracks of Illumina-based data and it happens that sometimes the center line is not placed at the actual SNP position, but one nucleotide aside. I could not make up any sense of the behavior because in a series of ctrl+f keystrokes it was sometimes correct and sometimes not. Moving then backwards with ctrl+b gave also varying results. I little bit suspected whether jump to a new chromosome reference gives the correct behavior and subsequent walk along the same chromosome will be off-by-one-placed, but it was not that.

Please retry yourself. Also using some IonTorrent-based BAM files which have INDELs. 

Background:

I recently communicated on this with Agilent SureCall developers and they had some bug in their 2.1.1.13 version but fixed it for 3.0.1.4. However, the off-by-one error with INDELs seems to be false alarm, as it is conforming the specs. However, I would be happy if you double-checked IGV behavior in this regard. It is not clear tom what they fixed and what was eventually a bug in the igv copy they have bundled in their software. That is why I went to test current IGV straight from BI website with mixed feelings.

<pre>
1. The viewer is centered at the position of the variant reported in the mutation table (POS field). 
2. SureCall determines this position as per the specifications of VCF v4.1. As per VCF v4.1 specifications, 
	For SNPs, this is the location where the alternate allele is detected(SNP is called).
	For indels, this is the base before (left to) the event.
</pre>

<pre>
... we verified that the track focusing issue reported by you in v2.1.1.13 has been fixed in v3.0.
To reiterate our previous discussion the position of the focus is dictated by the value in the Pos field.
As per the VCF file format (v4.1) specs which is adopted in SureCall and noticed that for insertions
and deletions, the Pos is one base left of the event and for SNPs, it is the location of the event.

In version 3.0 we noticed that the observations are in sync with this specification.
</pre>

**Comment by jrobinso** [comment]
Hi, the version numbers quoted don't make any sense, they are nor our version numbers.  Verify that you are testing IGV downloaded from www.broadinstitute.org/igv,  other entities have forked igv and have their own versions.

**Comment by mmokrejs** [comment]
That is why I asked you to check yourself. ;)

Based on the install logs it seems they are now serving us with:

Install File:             C:\Agilent\Agilent_SureCall_3.0.1.4\Client\lib\igv-2.1.jar
                          Status: SUCCESSFUL

No idea what modifications their 2.1 version has.


No matter what they have, please check how your 2.3.52 behaves. As I wrote, I still did observe weird jumps to an off-by-one position. Thank you.

**Comment by jrobinso** [comment]
Sorry but this doesn't even look like a bug to me, please don't open git 
issues on other peoples distributions of IGV,  if you feel there is a 
bug or have a feature request first verify the behavior with our 
distribution.

**Comment by jrobinso** [comment]
Also, BTW, 2.1 is absolutely ancient.

**Comment by mmokrejs** [comment]
As I already wrote, it is an issue with current browser. I just mentioned we found it with an old version and what was the bug about.

Anyway, with igv-2.3.52 on Linux I get center-line placed one nucleotide leftwards from the actual SNP position. See the below figure. This happens only when moving between SNP's using ctrl+f AND while you jump to a new chromosome. Additional jumps ALONG the "current" chromosome do not have this problem. In the figure below, the center line is at 6,404,280 whereas VCF is placed at 6,404,281 and indeed, the A's are at 6,404,281. So, only the center-line placement is wrong.

![center_line_is_at_location_6404280_instead](https://cloud.githubusercontent.com/assets/3297852/7630593/d8f0b388-fa37-11e4-9d90-32c4e4d6cdd2.png)
